packages/react-storage/src/components/StorageBrowser/views/LocationsView/Controls/DataTable.tsx

```diff
@@ -101,37 +110,49 @@ const getLocationsData = ({
     }
   }
 
-  const rows = data.map((location, index) => [
-    {
-      key: `td-name-${index}`,
-      children: (
-        <ButtonElement
-          className={TABLE_DATA_BUTTON_CLASS}
-          onClick={() => onLocationClick(location)}
-          variant="table-data"
-        >
-          {location.scope}
-        </ButtonElement>
-      ),
-    },
-    { key: `td-type-${index}`, children: location.type },
-    { key: `td-permission-${index}`, children: location.permission },
-  ]);
+  const rows = data.map((location, index) => {
+    const parsedLocation = parseLocationAccess(location);
+    const row = [
+      {
+        key: `td-name-${index}`,
+        children: (
+          <ButtonElement
+            className={TABLE_DATA_BUTTON_CLASS}
+            onClick={() => onLocationClick(location)}
+            variant="table-data"
+          >
+            {parsedLocation.prefix
+              ? parsedLocation.prefix
+              : `${parsedLocation.bucket}/`}
+          </ButtonElement>
+        ),
+      },
+      { key: `td-bucket-${index}`, children: parsedLocation.bucket },
+      { key: `td-type-${index}`, children: location.type },
+      { key: `td-permission-${index}`, children: location.permission },
+    ];
+    if (hideType) {
+      row.splice(2, 1);
+    }
+    return row;
+  });
 
   return { columns, rows };
 };
 
 export function DataTableControl({
   range,
+  hideType = false,
```

Comment:
do we need an option or should we just remove it by default? Do we anticipate adding it back?

Reply:
Ah, I didn't recognize that this table was only used for the locations view. Yes, just removing type altogether makes the most sense.

Reply:
Removed both `hideType` and usage of type in the table